cargo-near/src/build.rs

```diff
@@ -47,27 +48,24 @@ pub(crate) fn run(args: BuildCommand) -> anyhow::Result<()> {
         pretty_abi_path.replace(util::copy(&path, &out_dir)?);
 
         if args.embed_abi {
-            step_printer.print_step("Compressing ABI to be embedded");
-            let AbiResult { path } = abi::write_to_file(
-                &contract_abi,
-                &crate_metadata,
-                AbiFormat::JsonMin,
-                AbiCompression::Zstd,
-            )?;
+            let path = util::handle_step("Compressing ABI to be embedded..", || {
+                let AbiResult { path } = abi::write_to_file(
+                    &contract_abi,
+                    &crate_metadata,
+                    AbiFormat::JsonMin,
+                    AbiCompression::Zstd,
+                )?;
+                anyhow::Ok(path)
+            })?;
             min_abi_path.replace(util::copy(&path, &out_dir)?);
-        } else {
-            step_printer.print_step("Skipped embedding");
         }
     }
 
     if let (true, Some(abi_path)) = (args.embed_abi, &min_abi_path) {
         cargo_args.extend(&["--features", "near-sdk/__abi-embed"]);
         build_env.push(("CARGO_NEAR_ABI_PATH", abi_path.to_str().unwrap()));
-        step_printer.print_step("Building contract (with embedded ABI)");
-    } else {
-        step_printer.print_step("Building contract");
     }
-
+    util::print_step("Generating ABI");
```

Comment:
```suggestion
    util::print_step("Building Contract");
```

Reply:
Just realized in the comment above I commended you on standardizing the case, then went ahead to violate that in my this comment 😆

Reply:
Ahahaha, I didn't even notice myself